data/json/overmap/specials.json

```diff
@@ -2406,8 +2406,8 @@
     ],
     "locations": [ "wilderness" ],
     "city_distance": [ 1, 6 ],
-    "city_sizes": [ 4, 12 ],
-    "occurrences": [ 0, 5 ],
+    "city_sizes": [ 4, 20 ],
+    "occurrences": [ 0, 3 ],
```

Comment:
TBH even 3 zoos in a single overmap seems very excessive. I think this could go do 0,1, and the minimum city size could get bumped up a bit as well.